docs/architecture/diagrams/models_diagram.md

```diff
@@ -28,60 +38,147 @@ class "registrar.Contact <Registrar>" as registrar.Contact #d6f4e9 {
     + created_at (DateTimeField)
     + updated_at (DateTimeField)
     ~ user (OneToOneField)
-    + first_name (TextField)
-    + middle_name (TextField)
-    + last_name (TextField)
-    + title (TextField)
-    + email (TextField)
+    + first_name (CharField)
```

Comment:
This whole section is autogenerated, dont review from here to the end of the file as this is just the command output